test/unit/spec/PasswordReset_spec.js

```diff
@@ -152,16 +170,40 @@ function (Q, _, $, OktaAuth, LoginUtil, Util, PasswordResetForm, Beacon, Expect,
         expect(test.form.subtitleText()).toEqual('Your password must have a symbol.');
       });
     });
+
     itp('has a valid subtitle if only password complexity "excludeUsername" defined', function () {
       return setup({policyComplexity: 'excludeUsername'}).then(function (test) {
         expect(test.form.subtitleText()).toEqual('Your password must have no parts of your username.');
       });
     });
+
     itp('has a valid subtitle if only password age "history" defined', function () {
       return setup({policyAge: 'history'}).then(function (test) {
         expect(test.form.subtitleText()).toEqual('Your password cannot be any of your last 7 passwords.');
       });
     });
+
+    itp('has a valid subtitle in days if only password age "minAgeInDays" defined', function () {
+      return setup({policyAge: 'minAgeInDays'}).then(function (test) {
+        expect(test.form.subtitleText()).toEqual(
+          'At least 2 day(s) must have elapsed since you last changed your password.');
+      });
+    });
+
+    itp('has a valid subtitle in hours if only password age "minAgeInHours" defined', function () {
+      return setup({policyAge: 'minAgeInHours'}).then(function (test) {
+        expect(test.form.subtitleText()).toEqual(
+          'At least 2 hour(s) must have elapsed since you last changed your password.');
+      });
+    });
+
+    itp('has a valid subtitle in minutes if only password age "minAgeInMinutes" defined', function () {
+      return setup({policyAge: 'minAgeInMinutes'}).then(function (test) {
+        expect(test.form.subtitleText()).toEqual(
+          'At least 30 minute(s) must have elapsed since you last changed your password.');
+      });
+    });
+
```

Comment:
we need to add 2 more tests
- minAge combined with historyCount
- all options (```{policyComplexity: 'all', policyAge: 'all'}``` (or something similar)